.github/workflows/release.yml

```diff
@@ -41,10 +41,10 @@ jobs:
 
       - name: Publish artifacts for all Scala versions
         env:
-          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-          # note: although the sbt-bintray plugin expects a var name BINTRAY_PASS, we must pass a bintray key
-          BINTRAY_PASS: ${{ secrets.BINTRAY_KEY }}
-        run: sbt "+publish"
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+        run: sbt ci-release
```

Comment:
Oh, `ci-release` does the cross-publishing already? No need to `+ci-release`. Hmmm, OK.